Readme.md

```diff
@@ -0,0 +1,175 @@
+
+<a name="readme-top"></a>
+<div align="center">
+  
+  <img src="projectLogo.png" alt="logo" width="340"  height="auto"/>
+  <br/>
+
+  <h3><b>Finish Portfolio Setup Mobile version</b></h3>
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
[OPTIONAL] Well done on the project 🎉 but please make sure that your README contains all of the sections described in the original template as they are required (except for the FAQ). 😉